### PR TITLE
Update .pre-commit-config.yaml

### PR DESCRIPTION
**Pull request by juancarlospaco**
## What kind of change does this PR introduce?

- Update `.pre-commit-config.yaml` to use the latest versions of hooks.

### CODE REVIEW

**Review by J0 — APPROVED** [review]
Thanks! LGTM. Please wait for CI to pass before merging though

**Review by silentworks — APPROVED** [review]

**Pull request merged**
